homeassistant/components/xiaomi_miio/config_flow.py

```diff
@@ -284,7 +299,12 @@ async def async_step_cloud(
             return await self.async_step_select()
 
         return self.async_show_form(
-            step_id="cloud", data_schema=DEVICE_CLOUD_CONFIG, errors=errors
+            step_id="cloud",
+            data_schema=DEVICE_CLOUD_CONFIG,
+            errors=errors,
+            description_placeholders={
+                "cloud_servers_url": "https://www.openhab.org/addons/bindings/miio/#country-servers"
```

Comment:
For step `cloud`, the url is provided 3 times.
Please create and use a local constant (eg. `CLOUD_SERVERS_URL`)
```suggestion
                "cloud_servers_url": CLOUD_SERVERS_URL
```